chapters/derivationofstream.tex

```diff
@@ -195,7 +195,7 @@ \subsection{Connection of 3 stream connectors where one mass flow rate is identi
 properties discussed for two connected components still hold. The
 connection set equations reflect that the sensor does not any influence
 by discarding the flow rate of the latter. In several cases a non-linear
-equation system is removed by this transformation. However, inStream(..)
+equation system is removed by this transformation. However, \lstinline!inStream!(..)
```

Comment:
and here

Reply:
Strange with the two dots though

Reply:
Also issues in the original word-document. (Including the two dots). 

There are about 44 cases of (..) in Modelica 3.4. 
In some cases it may be intentional since fixed-width dots make `(...)` look too wide - however, that can be adjusted to use proper ldots.

Reply:
I'm open to fix all these, but before doing so we would need a guideline from MAP-LANG, how the specification should look like. 

In many cases we did intentionally do a "stupid robot job", because there was no consistent way to be seen, so we "copied" the word document.

Reply:
Thanking you for spotting. Fixed.

Regarding consistency and general looks: I did some minor improvements, but main task was to get the LaTeX document up to the state of Word. See my initial description of the pull request: "Syntax highlighting is still inconsistent in many places. But, at least we should have the same status/looks like in version 3.4, again."